fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -815,13 +815,7 @@ private void initAggregateBuiltins() {
                 Type.VARCHAR, Type.VARCHAR, true, false, false));
 
         for (Type t : Type.getSupportedTypes()) {
-            if (t.isNull()) {
-                continue; // NULL is handled through type promotion.
-            }
-            if (t.isChar()) {
-                continue; // promoted to STRING
-            }
-            if (t.isTime()) {
+            if (t.isNull() || t.isChar() || t.isTime() || t.isArrayType() || t.isJsonType() || t.isPseudoType()) {
```

Comment:
Why not replace it with `isNotNumeric`?

Reply:
Yes, it's better

Reply:
We can apply window function on `DATETIME` which is not numerical, although it does stored in numerical format